tests/test_extractsegmentationextractor.py

```diff
@@ -155,3 +161,20 @@ def test_extractor_accepted_list(self, accepted_list):
             self.extractor.get_rejected_list(),
             list(set(range(20)) - set(accepted_list)),
         )
+
+    def test_extractor_config_dict(self):
+        """Test that the extractor class reads the config struct correctly."""
+
+        with h5py.File(self.test_config_path + "/test_file.mat", "a") as test_config_struct:
+            thresholds = test_config_struct.create_group("thresholds")
+            test_config_struct.create_dataset("S_corr_thresh", data=np.array([0.1]))
+            test_config_struct.create_dataset("max_iter", data=np.array([10.0]))
+            thresholds.create_dataset("S_dup_corr_thresh", data=np.array([0.95]))
+            thresholds.create_dataset("T_dup_corr_thresh", data=np.array([0.95]))
+
+            config = self.extractor._config_struct_to_dict(config_struct=test_config_struct)
+
+        self.assertEqual(config["S_corr_thresh"], [0.1])
+        self.assertEqual(config["max_iter"], [10.0])
+        self.assertEqual(config["thresholds"]["S_dup_corr_thresh"], [0.95])
+        self.assertEqual(config["thresholds"]["T_dup_corr_thresh"], [0.95])
```

Comment:
@CodyCBakerPhD is this a good approach here, or should I return values that exactly match [ndx-extract](https://github.com/catalystneuro/ndx-extract) expected input?